cscs-checks/apps/lammps/lammps_check.py

```diff
@@ -31,6 +36,15 @@ def __init__(self, name, **kwargs):
             }
         }
 
+    def setup(self, partition, environ, **job_opts):
+        self.sanity_patterns = sn.all([
+            sn.assert_found(r'Total wall time:', self.stdout),
+            sn.assert_reference(self.energy,
```

Comment:
I have a concern about how suitable the `assert_reference` is for numerical sanity checking. We usually deal with absolute values and not percentages. @victorusu What do you think?

Reply:
I agree with @vkarak. I think we should not use percentages.